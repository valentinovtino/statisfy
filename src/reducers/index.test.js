import rootReducer from './index.js';

describe('rootReducer', () => {

  it('matches snapshot', () => {
    expect(rootReducer).toMatchSnapshot();
  });
});