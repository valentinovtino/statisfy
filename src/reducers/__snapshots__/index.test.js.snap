// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rootReducer matches snapshot 1`] = `undefined`;
